### DISCUSSION TITLE
PaddleOCR/latest/ppocr/blog/whl

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14265] [Q&A]
# PaddleOCR/latest/ppocr/blog/whl

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/blog/whl.html

**Comment by forgeyao** [comment]
文档没更新。有大量参数不在文档里，已有的参数也不是最新的，比如 ocr_version 默认已经是 PP-OCRv4

**Reply by GreatV** [reply]
是的，目前paddleocr社区精力有限，还没有更新文档，如果你对这块感兴趣，欢迎提交PR更新相关文档。

**Comment by Dien666** [comment]
识别pdf的时候报错，ImportError: Failed importing fitz. This likely means that some paddle modules require additional dependencies that have to be manually installed (usually with `pip install fitz`). 但手动pip之后依然报这个错

**Reply by GreatV** [reply]
试试 `pip install pymupdf`

**Reply by Essence9999** [reply]
> 试试 `pip install pymupdf`

@GreatV 你好，想请教个问题：有个疑问pymupdf的license是AGPL 3.0，而本项目是Apache-2.0 license，是否有冲突？

**Reply by GreatV** [reply]
@Essence9999 有冲突，现在默认不安装。
